### PR TITLE
Add custom icon set for perf. Fixes #38

### LINKED ISSUE #38: Remove iron-icons import
I noted that this element imports the full iron-icons set for the arrow-drop-down icon. As I watch [polycast](https://www.youtube.com/watch?v=tjmRUgUca1g&list=PLOU2XLYxmsII5c3Mgw6fNYCzaWrsM3sMN&index=3), this is bad for the performance. Maybe the iron-icons import should be removed for that reason.

### PR DESCRIPTION
**Pull request by ebidel**
R: @notwaldorf

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -24,7 +24,6 @@
     "polymer": "polymer/polymer#^1.1.0",
     "iron-a11y-keys-behavior": "polymerelements/iron-a11y-keys-behavior#^1.0.0",
     "iron-icon": "polymerelements/iron-icon#^1.0.0",
-    "iron-icons": "polymerelements/iron-icons#^1.0.0",
```
Comment (notwaldorf) [inline bower.json]: need to add `iron-iconset-svg` as a dep instead

Comment (ebidel) [inline bower.json]: Done

**Comment by ebidel** [comment]
PTAL

File: bower.json (hunk 1 above)
Comment (notwaldorf) [inline bower.json]: There is no `iron-iconset-svg#^1.1.3`. Do we need something particular in that release, or does the classic `#^1.0.0` work?

**Comment by ebidel** [comment]
ping @notwaldorf

**Comment by notwaldorf** [comment]
@ebidel I left a note -- your `bower.json` is wrong

File: bower.json (hunk 1 above)
Comment (ebidel) [inline bower.json]: Whoops. Read this element's version and did a silly copy and paste. Updated.

**Comment by ebidel** [comment]
PTAL

**Comment by notwaldorf** [comment]
@ebidel OMG now it needs a rebase

**Comment by ebidel** [comment]
Rebased

**Comment by notwaldorf** [comment]
@ebidel The tests are green on master but red here, and this worries me :)

**Comment by miztroh** [comment]
@notwaldorf Looks like they just timed out.  Try running them again.

**Pull request merged**